astropy/tests/plugins/display.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This plugin provides customization of the header displayed by pytest for
+reporting purposes.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import os
+import sys
+import datetime
+import locale
+import math
+from collections import OrderedDict
+
+from ...extern import six
+
+from ..helper import ignore_warnings
+from ...utils.introspection import resolve_name
+
+
+PYTEST_HEADER_MODULES = OrderedDict([('Numpy', 'numpy'),
+                                     ('Scipy', 'scipy'),
```

Comment:
Even if this stays here as a plugin, I would suggest to remove most of the entries from the dictionary, and keep only numpy and maybe mpl?

Reply:
Maybe these should be configurable using `setup.cfg`?

Reply:
+1 to configuring via setup.cfg

Reply:
@astrofrog, @bsipocz I think I will leave this alone until we move this logic to `pytest-astropy`.